hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/tables/containersTable.tsx

```diff
@@ -0,0 +1,258 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useRef } from 'react';
+import moment from 'moment';
+import filesize from 'filesize';
+import Table, {
+  ColumnProps,
+  ColumnsType,
+  TablePaginationConfig
+} from 'antd/es/table';
```

Comment:
Additionally, there seems to be an issue with the new UI overview page. When we click the "View More" option in overview page to redirect to the containers page, it incorrectly redirects us to a "`/missingContainers`" page, which does not exist.

<img width="1662" alt="image" src="https://github.com/user-attachments/assets/1dfa1e4d-f1b7-4f49-a343-fe3e3f9acf60">

This issue does not occur when there are no unhealthy container records on the containers page.

<img width="501" alt="image" src="https://github.com/user-attachments/assets/f0304a43-c927-41a9-8e9c-8c45fc610352">

Reply:
Thanks for pointing this out, it was to be added as a part of this PR as we didn't have the container page finalized.
Made the changes as a part of [3f9953c](https://github.com/apache/ozone/pull/7267/commits/3f9953c0fbbd2175ee83e8f0b4927e45e9c10ac1)